lcrs_embedded/system/cpu_load.py

```diff
@@ -0,0 +1,51 @@
+"""
+Generate a load on the CPU with one process per core.
+"""
+import logging
+from multiprocessing import Process
+import time
+
+logger = logging.getLogger(__name__)
+
+def timed_cpu_load(endTime):
+    """
+    Add some numbers for a while.
+    Could have been placed inside cpu_load below but there it
+    is called within a separate process and is then not picked
+    up in test coverage. Solved with a direct call in the test.
+    """
+    m = 1
+    running = True
+    while running:
+        m = (m + 1) % 100000
+        running = (m != 0) or (time.time() < endTime)
```

Comment:
Can the whole thing not be replaced by:

```python
while time.time() < end_time:
    m = (m + 1) % 100000
    # This can be added if you need a forced evaluation of m in case
    # it's garbage collected in advance - but I don't think Python would
    # do this.
    str(m)
```

Oh small thing... camelCase is mostly never used in Python for variables, only for ClassNames :)